microsetta_interface/templates/account_details.jinja2

```diff
@@ -163,28 +189,36 @@
                 <label for="post_code" id="post_code_label"></label>
                 <input id="post_code" name="post_code" class="form-control" value="{{account.address.post_code |e}}" type="text" />
             </div>
+            <div class="form-group">
+                <label for="language" name="language_label">{{ _('Preferred language for future communications') }}*</label>
+                <br>
+                <select id="language" name="language">
+                    <option value="en_US" {% if account.language == "en_US" %} selected {% endif %} >English</option>
```

Comment:
I think push beyond this PR to avoid scope creep.  Once #173 is in, this particular line will come for free with a change to model_i18n, but until we have a separate .PO file, the difference is pretty meaningless.